.pre-commit-config.yaml

```diff
@@ -0,0 +1,51 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.0.1
+  hooks:
+  - id: check-added-large-files
+  - id: check-toml
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+- repo: https://github.com/psf/black
+  rev: 21.10b0
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.910-1
+  hooks:
+  - id: mypy
+    exclude: ^(docs/|example-plugin/|tests/fixtures)
+- repo: local
+  hooks:
+  - id: pytest
```

Comment:
it can't be used because it use duty

Reply:
Not sure what duty is, I'll just comment it out

Reply:
https://github.com/pawamoy/duty

but it require duties.py on project